README.md

```diff
@@ -90,6 +90,12 @@ You can also spin up *LocalStack* without any installation requirements, using D
 docker run -it -p 4567-4580:4567-4580 -p 8080:8080 atlassianlabs/localstack
```

Comment:
Can we change the port range to `4567-4581:4567-4581` here? (that was an oversight in a previous commit)